components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -181,12 +182,17 @@ export class GithubApp {
             options.getRouter("/reconfigure").get(
                 "/",
                 asyncHandler(async (req: express.Request, res: express.Response) => {
-                    const gh = await app.auth();
-                    const data = await gh.apps.getAuthenticated();
-                    const slug = data.data.slug;
-
-                    const state = req.query.state;
-                    res.redirect(`https://github.com/apps/${slug}/installations/new?state=${state}`);
+                    try {
```

Comment:
todo: extract in simple PR to log the actual error *and* stop rending the stack trace